diff -ruN orig/SConscript new/SConscript
--- orig/SConscript	2019-08-14 14:28:06.000000000 -0600
+++ new/SConscript	2019-08-16 15:24:33.103935739 -0600
@@ -366,7 +366,7 @@
                 unix_bin_path = os.path.join(install_root,
                     env.get('BINDIR', default='bin'))
                 unix_lib_path = os.path.join(install_root,
-                    env.get('LIBDIR', default='lib'))
+                    env.get('LIBDIR', default='lib64'))
 
                 binary = env.Install(unix_bin_path, binary_files)
                 skins = env.Install(os.path.join(unix_share_path, 'mixxx', 'skins'), skin_files)
